utils/test_utils.py

```diff
@@ -0,0 +1,259 @@
+# Copyright (c) 2021
+# MKS Plugin is released under the terms of the AGPLv3 or higher.
+
+# To run unit tests:
+# 1. install pytest: https://docs.pytest.org/en/latest/getting-started.html
+# 2. copy Uranium (https://github.com/Ultimaker/Uranium/tree/master/UM) to python libs folder (ex.: C:\Program Files\Python310\Lib)
+# 3. - run tests from command line: python -m pytest
+#    - run tests from Visual Studio Code: https://code.visualstudio.com/docs/python/testing
+
+import utils
+
+def test_generate_new_filename_abcd_30():
```

Comment:
It's hard to understand difference between **test_generate_new_filename_abcd_30** and **test_generate_new_filename_abc_10** when you read test name. 
I think you can name it more human-readable, f.ex. test_generate_new_filename_without_truncation , test_generate_new_filename_with_truncation_after_10_characters

Reply:
renamed test names, thanks. Agree, now they are more readable